### ISSUE TITLE
Trigger based on MATT's

### ISSUE THREAD

**Issue by WinterrrWarlock** [napolitanod/Danger-Zone#114]
Would it be possible to add a trigger condition from Monk's Active Tile Triggers?

I have a ship map that has the crew fighting off a Kraken. Instead of their usual turn, they instead have to fire cannons at the Kraken. I'd love to be able to trigger Danger zone automatically within a landing when they succeed, setting off the canon fire effects as well as damaging the Kraken. I can't really use the combat triggers because in some rounds they have to load the cannons. If there is a way to trigger the Danger Zone within a MATT workflow that would be awesome

**Comment by WinterrrWarlock** [comment]
Also, bit of a sidebar, but whenever I try to activate healing it instead does damage. I have to put the roll as -1d12 for it to "heal"

**Comment by napolitanod** [comment]
If MATT can call a macro or script, you can call the dangerZone API to trigger the zone. See https://github.com/napolitanod/Danger-Zone/wiki/API#triggerzone-zonename-sceneid-options--

For example, here is how I call a zone titled "Hand Grenade" - I hard coded the location for illustration purposes, but you can pass in things like the target token x and y for the coordinates. 

The api can be called from things like ItemMacro if you have familiarity with that. 

```js
 await dangerZone.triggerZone (
      "Hand Grenade", 
       canvas.scene.id, 
      {
      location: {x: 1000, y: 1000, z: 0}
      }
)
      ```

**Comment by napolitanod** [comment]
Thanks for the info on healing. I'll check out what's going on there.

**Comment by napolitanod** [comment]
The issue with healing doing damage is fixed with 1.9.6, which I just pushed out now. Thanks again for reporting!
